_currencies.md

```diff
@@ -1,5 +1,79 @@
 
-In our Knowledge Base you will always find the most recent and up-to-date [list of the currencies we support](https://support.uphold.com/hc/en-us/articles/202473803), along with the three letter abbreviation you can use when referring to the currency within the API.
+<aside class="notice">
+  For brevity and convenience, we refer to any such asset simply as "currency" throughout the API documentation.
+  In cases that only apply to specific classes of assets, we will clarify accordingly.
+</aside>
+
+In Uphold's API, various endpoints include currencies in their input or output. We represent all such currencies by an abbreviation **code** of variable length, typically containing uppercase letters (for example, a currency's [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) code, an equity's [ticker symbol](https://en.wikipedia.org/wiki/Ticker_symbol), or other similar well-known representation).
+
+## List Supported Assets
+
+> Example of unauthenticated request to list all supported assets:
+
+```bash
+curl https://api.uphold.com/v0/assets
+```
+
+> The above command returns JSON in the following format:
+
+```json
+[
+  {
+    "code": "BTC",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 8
+    },
+    "name": "Bitcoin",
+    "status": "open",
+    "symbol": "฿",
+    "type": "cryptocurrency"
+  },
+  {
+    "code": "USD",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 2
+    },
+    "name": "US Dollar",
+    "status": "open",
+    "symbol": "$",
+    "type": "fiat"
+  }
+]
+```
+
+> Example of authenticated request to list only the assets available to the current user:
+
+```bash
+curl https://api.uphold.com/v0/assets \
+  -H "Authorization: Bearer <token>"
+```
+
+Get the list of supported currencies and other financial assets, with details on their name, type, [status](#asset-status), and formatting specification for user-facing display.
+
+If the request is unauthenticated, the full list of assets supported by Uphold is returned.
+Authenticated requests, on the other hand, will filter the output, returning only the assets available for the current user, which can depend on factors such as their country and state.
+
+Please note that this endpoint is [paginated](#pagination), so multiple requests may be required to get the complete list.
+
+### Asset status
+
+The `status` field of an asset indicates whether it can be transacted at the moment.
+The possible values are:
+
+* `open` — the asset is in regular operation and transactions can be made with it.
+* `closed` — the asset is temporarily disabled; typically used for equities
+  outside the U.S. stock market's operating hours, which are 9:30am to 4:30pm ET.
```

Comment:
The stock market also only operates on weekdays, except on bank holidays. Also there are some half-days thrown into the mix.

Reply:
Good point. I don't think we need to be super precise here, though. Maybe something like this would work?

> ...the U.S. stock market's operating hours, which are typically 9:30am to 4:30pm ET on weekdays.

Reply:
Works for me. You could add some links :smile: 
ex:
https://www.nyse.com/markets/hours-calendars
https://www.nasdaq.com/stock-market-trading-hours-for-nasdaq

Also, it says it closes at 4pm not 4:30pm

Reply:
Yeah, using links is a good solution. 👍
I'll investigate the source of the discrepancy.

Reply:
@Americas indeed the information I had previously been given had a typo. I've updated the docs accordingly, and also made the changes discussed above.